### ISSUE TITLE
Support ADA Compliance

### ISSUE THREAD

**Issue by paustria** [juliangarnier/anime#569]
**Is your feature request related to a problem? Please describe.**
Currently animejs using `transform: translateX`. This causes an issue with the accessibility 
 reader tool. I use https://www.nvaccess.org/download/ to test it. I tested on Firefox, windows machine.

For example,
https://codepen.io/anon/pen/YMLgbY
When I hover the text, NVDA reader does not read the text.

**Describe the solution you'd like**
The accessibility tool should read the text when the mouse is hovering on the element.

**Describe alternatives you've considered**
I might have to use animate.css or create custom CSS instead.

**Additional context**
Please consider this feature. I make the site that requires to support ADA. I have to remove animejs out of my project because this feature is not supported yet.

**Comment by kn0wn** [comment]
Hi @paustria, I'd like to know more about this issue. Please can you explain how using custom CSS solves this issue and we may be able to implement something into animejs.

**Comment by paustria** [comment]
@kn0wn Thank you for consider this. I'm currently busy with my tasks at work. It has been awhile and I already changed job. Please allow me to reproduce this again this weekend.
